riscof/main.py

```diff
@@ -291,7 +312,13 @@ def execute():
 
         with open(platform_file, "r") as platfile:
             pspecs = platfile.read()
-
+            
+        if args.dbfile is not None and args.testfile is not None:
```

Comment:
Instead of manually checking and handling errors for exclusivity, it would be better if you use the [mutually exclusive group](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_mutually_exclusive_group) in argparse to ensure exclusivity. That way the parser can enforce exclusivity.

Reply:
Sure

Reply:
Done